dart/lib/src/sentry_tracer.dart

```diff
@@ -100,7 +108,7 @@ class SentryTracer extends ISentrySpan {
         }
       });
 
-      final transaction = SentryTransaction(this);
+      final transaction = SentryTransaction(this, measurements: _measurements);
```

Comment:
Instead of passing via ctor, I'd rather call `transaction.addMeasurements(...)`, the reason is, we'd not need an optional ctor param for `SentryTransaction` only for that.

Reply:
If going down that route we always have to init the instance, since the class is marked as `@immutable` and we'd also have do do additional work when calling the copy method.

Reply:
We do need it as events are copied when going through processing and we'd loose measurements otherwise.